tests/functional/pages/components/radio/mandatory_with_mandatory_other_overridden_error/summary.page.js

```diff
@@ -0,0 +1,19 @@
+// >>> WARNING THIS PAGE WAS AUTO-GENERATED - DO NOT EDIT!!! <<<
+const QuestionPage = require('../../../surveys/question.page);
+
+class SummaryPage extends QuestionPage {
+
+  constructor() {
+    super('summary');
+  }
+
+  radioMandatoryAnswer() { return '#radio-mandatory-answer-answer'; }
+
+  radioMandatoryAnswerEdit() { return '[data-qa="radio-mandatory-answer-edit"]'; }
+
+  otherAnswerMandatory() { return '#other-answer-mandatory-answer'; }
+
+  otherAnswerMandatoryEdit() { return '[data-qa="other-answer-mandatory-edit"]'; }
+
+}
+module.exports = new SummaryPage();
```

Comment:
Why is this red ?

Reply:
Imma just leave this here: https://stackoverflow.com/questions/30667675/what-does-a-red-text-background-mean-in-github-comparison-view maybe just git being a git

Reply:
Because it missing closing quote on line 2 `require('../../../surveys/question.page);`

Reply:
Yh my latest push should have fixed this?